Compiled from "ThreadLocalRandom.java"
final class io.netty.util.internal.ThreadLocalRandom$2 implements java.lang.Thread$UncaughtExceptionHandler {
  io.netty.util.internal.ThreadLocalRandom$2();
    Code:
       0: aload_0
       1: invokespecial #16                 // Method java/lang/Object."<init>":()V
       4: return

  public void uncaughtException(java.lang.Thread, java.lang.Throwable);
    Code:
       0: invokestatic  #24                 // Method io/netty/util/internal/ThreadLocalRandom.access$200:()Lio/netty/util/internal/logging/InternalLogger;
       3: ldc           #26                 // String An exception has been raised by {}
       5: aload_1
       6: invokevirtual #30                 // Method java/lang/Thread.getName:()Ljava/lang/String;
       9: aload_2
      10: invokeinterface #36,  4           // InterfaceMethod io/netty/util/internal/logging/InternalLogger.debug:(Ljava/lang/String;Ljava/lang/Object;Ljava/lang/Object;)V
      15: return
}
